drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -460,6 +460,13 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
 
 #endif /* MODULE_NETDEV_IEEE802154_OQPSK */
 
+        case NETOPT_RANDOM:
+            assert(max_len >= sizeof(uint32_t));
+            at86rf2xx_disable_smart_idle(dev);
+            at86rf2xx_get_random(dev, (uint8_t*)val, sizeof(val));
+            at86rf2xx_enable_smart_idle(dev);
```

Comment:
you are right, this is only used when the call to àt86rf2xx_get_random()` is required, this will require a small change in its parameters, I removed the const assignment from the `at86rf_2xx_t dev` parameter for the SRT mode variation